packages/backend/package.json

```diff
@@ -45,6 +45,7 @@
     "@backstage/plugin-techdocs-backend": "^1.9.6",
     "@roadiehq/scaffolder-backend-module-utils": "^1.13.2",
     "app": "link:../app",
+    "backstage-plugin-dask-onboarding-backend": "^0.1.3",
```

Comment:
This plugin should probably reside under `@kartverket`

Reply:
You need access to the scope on npmjs.com. I'll contact a couple of people and see if we can get that figured out :slightly_smiling_face:

Reply:
Nice, thank you! If this can take some time, could we merge this, and update the package later? The packages is published under the `datplattform-kartverket` user, see [this link](https://www.npmjs.com/settings/dataplattform-kartverket/packages)

Reply:
Absolutely, if we don't get a resolution by today I'll approve :slightly_smiling_face: